pages/events/sprints.vue

```diff
@@ -1,11 +1,73 @@
 <template>
-    <p>Events springs</p>
+    <i18n-page-wrapper class="px-8 sm:px-10 md:px-32 lg:px-60" custom-x>
+        <core-h1 :title="$t('title')"></core-h1>
+        <i18n path="intro" tag="p" class="intro"></i18n>
```

Comment:
should add `white-space: pre-line;` attribute for line-beak?

<details open>
<summary>Like this</summary>
<br>

![image](https://user-images.githubusercontent.com/16368408/132984181-36be2193-c98d-4b90-b01e-579c7b75ace9.png)
</details>

---

<details open>
<summary>Original</summary>

![image](https://user-images.githubusercontent.com/16368408/132984548-e105bec3-2df9-42e3-87ee-644eb1fc7058.png)

<br>

Reply:
Nice catch. A new commit is added.